content/en/docs/contribute/new-content/overview.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Contributing new content overview
+linktitle: Overview
+content_template: templates/concept
+main_menu: true
+weight: 5
+---
+
+{{% capture overview %}}
+
+This section contains information all contributors should know before beginning.
+
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Contributing basics
+
+- Write Kubernetes documentation in markdown and build the Kubernetes site using [Hugo](https://gohugo.io/).
```

Comment:
nit: Markdown
nit: line 20, The instructions, `write and build`, do not match the tone of the content in this paragraph.
Possible ideas:
You can create Kubernetes documentation in Markdown using any source code editor.
The Kubernetes website is built by the static site generator [Hugo](...).

Line 21: This should be reworded. Could this sentence/section link to the page/content describing how to build the documentation, test your changes locally?